app/assets/v2/css/home.css

```diff
@@ -319,6 +427,13 @@ figure:hover::before {
   animation: rock_arrow ease-in-out 4s infinite;
 }
 
+.hide_on_mobile {
+  display: block;
+}
+.show_on_mobile {
+  display: none;
+}
```

Comment:
^ let's use bootstrap classes
both of those can we replaced with `d-none d-sm-block` sort of thing

Reply:
That was my first approach but for some reason doesn't have any effect on the images so I implemented in this way.